bundle/DependencyInjection/Compiler/AssetThemePass.php

```diff
@@ -31,7 +31,7 @@ public function process(ContainerBuilder $container)
         // Look for assets themes in bundles.
         foreach ($container->getParameter('kernel.bundles') as $bundleName => $bundleClass) {
             $bundleReflection = new \ReflectionClass($bundleClass);
-            $bundleViewsDir = dirname($bundleReflection->getFileName()) . '/Resources/public';
+            $bundleViewsDir = \dirname($bundleReflection->getFileName()) . '/Resources/public';
```

Comment:
This requirement was either turned off by Ibexa Code style or forced to be in use directive